widgetssdk/src/test/java/android/text/TextUtils.java

```diff
@@ -0,0 +1,7 @@
+package android.text;
+
+public class TextUtils {
```

Comment:
It would be nice to have some comment here explaining that his class is here for test mocking

Reply:
Good suggestion, thank you. Added a comment. 58cf5850598c10d1508fe585fd7f44c82a29679a